projects/ngx-d3timeline/src/lib/axis/time-axis/time-axis-tick-renderer.ts

```diff
@@ -12,7 +20,18 @@ export class TimeAxisTickRenderer implements TickRenderer {
     return this.scale.tickFormat()(tick);
   }
 
-  getTransform(tick: any): number {
-    return this.scale(tick);
+  getTransform(tick: any): string {
+    const axisOffset = this.scale(tick);
+    return this.optionsService.getTranslation(
+      axisOffset,
+      0,
```

Comment:
I think there is too much logic here. Would be better to keep getTranslation to only depend on a range value, the view and orientation. Use dx and dy to position the tick relative to the translation by adding dx and dy properties to the Tick model. I created #79 and #80 for this.

Reply:
See above reply to similar comment

Reply:
Would rather just remove the code and have it drawn incorrectly than keep the code in.